src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -591,4 +617,17 @@ protected function note($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * Fire the given event for the migration.
+     *
+     * @param  object $event
```

Comment:
You can type-hint this and in the method signature that it conforms to your contract.

Reply:
Heh. i even did the contract partially so that I could use it here. Guess I forgot.

Reply:
You still need to do this.